src/osd/scrubber/scrub_machine.cc

```diff
@@ -538,10 +538,8 @@ sc::result WaitReplicas::react(const GotReplicas&)
       return transit<PendingTimer>();
 
     } else {
-
-      // maps_compare_n_cleanup() will arrange for MapsCompared event to be
-      // sent:
       scrbr->maps_compare_n_cleanup();
+      post_event(MapsCompared{});
```

Comment:
Or simply transition here to WaitDigestUpdate

Reply:
Yep, good point.  Updating